### PR TITLE
change:

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard: Add associated issues, tickets, docs URL here.

## Overview

Describe what your Pull Request is about in a few sentences.

## Changes

Describe your changes and implementation choices. More details make PRs easier to review.

 - fix: token balance is not showing on BNB & AVAX l2 network
 - fix: added emergency swap when there toosmallsec balance.

## Testing

Describe how to test your new feature/bug fix and if possible, a step by step guide on how to demo this.

### CODE REVIEW

File: packages/boba/gateway/src/services/app.service.js [hunk 1]
```diff
@@ -244,7 +244,7 @@ class AppService {
       payload: {
         currency: L1_ETH_Address,
         addressL1: L1_ETH_Address,
-        addressL2: L2_BOBA_Address,
+        addressL2: L2_SECONDARYFEETOKEN_ADDRESS,
```
Comment (jemeza) [inline packages/boba/gateway/src/services/app.service.js:247]: Shouldn't this be the L2 address of Ethereum? I believe `L2_SECONDARYFEETOKEN_ADDRESS` is the address of Boba?

File: packages/boba/gateway/src/containers/Bridging/BridgeInput/EmergencySwap/index.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import styled from 'styled-components'
+import { LAYER } from 'util/constant'
+import { NETWORK } from 'util/network/network.util'
+import BN from 'bignumber.js'
+import { logAmount } from 'util/amountConvert'
+import { isEqual } from 'lodash'
+import { openAlert } from 'actions/uiAction'
+import { getETHMetaTransaction } from 'actions/setupAction'
+
+export const SwapContainer = styled.div`
```
Comment (jemeza) [inline packages/boba/gateway/src/containers/Bridging/BridgeInput/EmergencySwap/index.tsx]: we should make the style it's own file soon

File: packages/boba/gateway/src/services/app.service.js (hunk 1 above)
Comment (sk-enya) [reply packages/boba/gateway/src/services/app.service.js:247]: No, it's address of secondary token on Boba Network. like
`BNB token on BobaBNB network`
`AVAX token on Boba Fuji network`

Earlier this has been change by some one so issue got reported like balance discrepancies

File: packages/boba/gateway/src/containers/Bridging/BridgeInput/EmergencySwap/index.tsx (hunk 2 above)
Comment (sk-enya) [reply packages/boba/gateway/src/containers/Bridging/BridgeInput/EmergencySwap/index.tsx]: Hmm correct

**Review by jemeza — APPROVED** [review]
LGTM

**Pull request merged**